rpcserver.go

```diff
@@ -1264,6 +1269,15 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 			return nil, err
 		}
 
+		// Before we publish the transaction we make sure it won't
+		// violate our reserved wallet value.
+		err = wallet.CheckReservedValueTx(sweepTxPkg.SweepTx)
```

Comment:
iiuc, once we have >1 anchor channel, this will always fail (assuming the confs are set to included all UTXOs)?

another option is to change the definition of "all" to be total balance minus reservations. there are probably a lot of users who wouldn't expect an operation like this to begin failing just because they started using anchor channels. this at least gives the functionality of "send all unused onchain balance to this addr".

Reply:
Changed this to add the reserved value as a second output to the sweep tx.